### ISSUE TITLE
Reminders to classify requests being sent three times within an hour

### ISSUE THREAD

**Issue by RichardTaylor** [mysociety/alaveteli#3386]
We've a WhatDoTheyKnow user complain about being sent the 

>Was the response you got to your FOI request any good?

chasing email three times. 

I've just experienced it myself today too. In respect of the same request I received a reminder at: 07:34, 07:56, and 08:21

Looking at my emails I can see this happened in relation to a previous request for which I got reminders on the 2nd of June; it doesn't appear to have happened prior to that. 

The latest request in question was that at:
https://www.whatdotheyknow.com/request/chief_constables_council_13_14_j

**Comment by garethrees** [comment]
This sounds like something is touching the request `updated_at`.

**Comment by garethrees** [comment]
`InfoRequest.where_old_unclassified` is producing a different date in the query compared to the original `InfoRequest.find_old_unclassified`:

```sql
-- irb(main):023:0* InfoRequest.find_old_unclassified(:order => 'info_requests.id', :include => [:user], :age_in_days => 3);nil
-- [DEPRECATION] InfoRequest#find_old_unclassified will be removed in 0.26
-- [DEPRECATION] InfoRequest#old_unclassified_params will be removed in 0.26
InfoRequest Load (1624.9ms)  SELECT "info_requests".* FROM "info_requests" WHERE (awaiting_description = 't'
AND last_public_response_at < '2016-07-16 10:37:14.253200'
AND url_title != 'holding_pen'
AND user_id IS NOT NULL) ORDER BY info_requests.id
User Load (1084.2ms)  SELECT "users".* FROM "users" WHERE "users"."id" IN (266, 305,

-- irb(main):027:0> InfoRequest.where_old_unclassified(3).order('info_requests.id').includes(:user).all;nil
InfoRequest Load (1213.1ms)  SELECT "info_requests".* FROM "info_requests" WHERE (awaiting_description = 't'
AND last_public_response_at < '2016-07-19 10:41:00.469162'
AND url_title != 'holding_pen'
AND user_id IS NOT NULL) ORDER BY info_requests.id
User Load (1104.9ms)  SELECT "users".* FROM "users" WHERE "users"."id" IN (266, 305,
```

**Comment by garethrees** [comment]
Need to call `#days` on the int:

```sql
-- irb(main):034:0> InfoRequest.class_eval do
-- irb(main):035:1*   def self.where_old_unclassified(age_in_days=nil)
-- irb(main):036:2>     age_in_days =
-- irb(main):037:2*       if age_in_days
-- irb(main):038:3>         age_in_days.days
-- irb(main):039:3>       else
-- irb(main):040:3*         OLD_AGE_IN_DAYS
-- irb(main):041:3>       end
-- irb(main):042:2>
-- irb(main):043:2*     where("awaiting_description = ?
-- irb(main):044:3"           AND last_public_response_at < ?
-- irb(main):045:3"           AND url_title != 'holding_pen'
-- irb(main):046:3"           AND user_id IS NOT NULL",
-- irb(main):047:3*           true, Time.zone.now - age_in_days)
-- irb(main):048:2>   end
-- irb(main):049:1> end
-- => nil
-- irb(main):050:0> InfoRequest.where_old_unclassified(3).order('info_requests.id').includes(:user).all;nil
  InfoRequest Load (1212.1ms)  SELECT "info_requests".* FROM "info_requests" WHERE (awaiting_description = 't'
 AND last_public_response_at < '2016-07-16 10:49:51.353605'
 AND url_title != 'holding_pen'
 AND user_id IS NOT NULL) ORDER BY info_requests.id
  User Load (1056.8ms)  SELECT "users".* FROM "users" WHERE "users"."id" IN (266, 305,
```

**Comment by garethrees** [comment]
Fixed by #3386
